src/lib/http-server.ts

```diff
@@ -29,14 +30,20 @@ import { assertIsAddressInfo } from './helpers';
  */
 export class HttpServer {
   #server: Server;
+  sslEnabled = false;
 
   /**
    * Creates a new instance of HttpServer.
    *
    * @param {RequestListener} requestListener The function that will handle the server's requests.
+   * @param {ServerOptions} options Optional ServerOptions to start the server with https.
```

Comment:
@lbestftr I'm not sure we're willing to support all possible options of an https server now. How about creating our own type only exposing we just need right now?

i.e. add in types.ts something like 

```
export type HttpServerOptions = Pick<ServerOptions, 'key'> &
  Pick<ServerOptions, 'cert'>;`
```

Reply:
Great idea, fixed!